site-src/implementations.md

```diff
@@ -260,6 +262,9 @@ advanced routing, multi-cluster load balancing and more. See the docs to deploy
 [private or public Gateways][gke-gateway-deploy] and also [multi-cluster
 Gateways][gke-multi-cluster-gateway].
 
+The GKE Gateway controller passes all the core Gateway API conformance tests
```

Comment:
```suggestion
The GKE Gateway controller passes all the core Gateway API conformance tests for the GATEWAY_HTTP conformance profile.
```

Feel free to rephrase it as you prefer, but we should make clear that  we are talking about the GATEWAY-HTTP conformance profile.

Reply:
Done. Thank you.